src/backend/joanie/core/api/client/__init__.py

```diff
@@ -1100,6 +1100,12 @@ def get_queryset(self):
                 type=OpenApiTypes.UUID,
                 many=True,
             ),
+            OpenApiParameter(
+                name="from_batch_order",
+                description="Boolean value if we want contracts from batch orders or not",
+                required=False,
+                type=OpenApiTypes.BOOL,
```

Comment:
As the type is already here:
```suggestion
                description="Retrieve contracts links for batch orders",
                required=False,
                type=OpenApiTypes.BOOL,
```